src/libAtomVM/context.h

```diff
@@ -172,6 +172,13 @@ enum ContextMonitorType
     CONTEXT_MONITOR_RESOURCE,
     CONTEXT_MONITOR_LINK_REMOTE,
     CONTEXT_MONITOR_MONITORING_LOCAL_REGISTEREDNAME,
+    CONTEXT_MONITOR_ALIAS,
+};
+
+enum ContextMonitorAliasType
+{
+    CONTEXT_MONITOR_ALIAS_EXPLICIT_UNALIAS,
+    CONTEXT_MONITOR_ALIAS_DEMONITOR,
```

Comment:
Let's use PascalCase. See AVMCCS-N004 rule in our coding style guide.
(There is still some old code that has to be migrated to updated style).